acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/LoggingTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright contributors to Hyperledger Besu
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.tests.acceptance;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.hyperledger.besu.tests.acceptance.dsl.AcceptanceTestBase;
+import org.hyperledger.besu.tests.acceptance.dsl.node.BesuNode;
+
+import java.io.IOException;
+
+import org.junit.Test;
+
+public class LoggingTest extends AcceptanceTestBase {
+
+  @Test
+  public void testDefaultLoggingIsAtLeastInfo() throws IOException {
+    final BesuNode node = besu.createArchiveNode("logger");
```

Comment:
ah nice, I was thinking the previous incantation was creating logging when it shouldn't but it was the other way around.

Any insight into why this factory method creates logging whereas the other one didn't?
These ATs are still in the realm of magic for me....something about runCommand vs runNodeStart?

Reply:
It's magic to me too.  It looks like run command is just there for the help test.  It requires an argument and an empty argument was breaking stuff.

The more reliable way to check for logging is to create a real running instance, this seemed as good as any other.